src/sentry/apidocs/parameters.py

```diff
@@ -256,13 +268,76 @@ class ProjectParams:
         description="Defaults to true where the behavior is to alert the user on every new issue. Setting this to false will turn this off and the user must create their own alerts to be notified of new issues.",
     )
 
+    DYNAMIC_SDK_LOADER_OPTIONS = OpenApiParameter(
+        name="dynamicSdkLoaderOptions",
+        location="query",
+        required=False,
+        type=inline_serializer(
+            name="DynamicSDKLoaderOptionsSerializer",
+            fields={
+                "hasReplay": serializers.BooleanField(required=False),
+                "hasPerformance": serializers.BooleanField(required=False),
+                "hasDebug": serializers.BooleanField(required=False),
+            },
+        ),
+        description="""
+Configures multiple options for the Javascript Loader Script.
+- `Performance Monitoring`
+- `Debug Bundles & Logging`
+- `Session Replay`: Note that the loader will load the ES6 bundle instead of the ES5 bundle.
+```json
+{
+    "dynamicSdkLoaderOptions": {
+        "hasReplay": true,
+        "hasPerformance": true,
+        "hasDebug": true
+    }
+}
+```
+""",
+    )
+
+    IS_ACTIVE = OpenApiParameter(
+        name="isActive",
+        location="query",
+        required=False,
+        type=bool,
+        description="Activate or deactivate the client key.",
+    )
+
+    RATE_LIMIT = OpenApiParameter(
+        name="rateLimit",
+        location="query",
+        required=False,
+        type=inline_serializer(
+            name="RateLimitParameterSerializer",
+            fields={
+                "window": serializers.IntegerField(required=False),
+                "count": serializers.IntegerField(required=False),
+            },
+        ),
+        description="""
+Applies a rate limit to cap the number of errors accepted during a given time window. To disable
+entirely set `rateLimit` to null.
```

Comment:
I did some testing and found that also setting either `window` or `count` or both to 0 in the request body also disables rate limiting. Should I include this as a way to disable it or only document the `null` to turn it off?